### ISSUE TITLE
utf8 support

### ISSUE THREAD

**Issue by acoloss** [techouse/sqlite3-to-mysql#46]
**Is your feature request related to a problem? Please describe.**
Hi, I'm trying to convert my sqlite 3 to mysql (using a cron so i can use it for some analytics) however i get the error:

> 2022-08-11 17:26:59 ERROR    Character set 'utf8' unsupported
> Character set 'utf8' unsupported

**Describe the solution you'd like**
is it difficult to add support for this, and i'm happy to donate.

**Comment by techouse** [comment]
It should support it already. Are you using Python 2 by any chance?

**Comment by acoloss** [comment]
Thanks for the quick response.
I'm using 3.10.4

> xxxx:~/.local/bin$ python --version
> Python 3.10.4

note, i am running the binary from the above directory, could that be related?

I used the pip3 installer.

**Comment by techouse** [comment]
Hmm. Why are you running it like a script? Did you install it using `pip` into its own virtual environment or did you just git clone this repo?

**Comment by techouse** [comment]
Could be related to this https://stackoverflow.com/questions/73244027/character-set-utf8-unsupported-in-python-mysql-connector

**Comment by acoloss** [comment]
```
| software               | version                              |
|------------------------|--------------------------------------|
| sqlite3-to-mysql       | 1.4.15                               |
|                        |                                      |
| Operating System       | Linux 5.15.0-43-generic              |
| Python                 | CPython 3.10.4                       |
| MySQL                  | MySQL client not found on the system |
| SQLite                 | 3.37.2                               |
|                        |                                      |
| click                  | 8.0.3                                |
| mysql-connector-python | 8.0.30                               |
| pytimeparse            | 1.1.8                                |
| simplejson             | 3.17.6                               |
| six                    | 1.16.0                               |
| tabulate               | 0.8.10                               |
| tqdm                   | 4.64.0                               |

```

Yes i installed it with pip3 install.
I got an error about PATH not set for the local bin dir, but ignored and just ran from this directory.

**Comment by acoloss** [comment]
> 

this was the issue
Thanks

**Comment by techouse** [comment]
For reference I will leave this link here: https://dev.mysql.com/doc/relnotes/connector-python/en/news-8-0-30.html

**Comment by techouse** [comment]
Fixed in [v1.4.16](https://github.com/techouse/sqlite3-to-mysql/releases/tag/v1.4.16)

**Comment by techouse** [comment]
@acoloss just out of curiosity, did you happen to use MariaDB while you encountered this bug?

**Comment by techouse** [comment]
Looks like this was finally fixed in v8.0.32

https://github.com/mysql/mysql-connector-python/blob/trunk/CHANGES.txt#L40
